node/src/sub_lib/blockchain_bridge.rs

```diff
@@ -34,27 +36,33 @@ impl Debug for BlockchainBridgeSubs {
 }
 
 #[derive(Clone, PartialEq, Eq, Debug, Message)]
-pub struct ReportAccountsPayable {
+pub struct RequestAvailableBalancesForPayables {
     pub accounts: Vec<PayableAccount>,
     pub response_skeleton_opt: Option<ResponseSkeleton>,
 }
 
-impl SkeletonOptHolder for ReportAccountsPayable {
+impl SkeletonOptHolder for RequestAvailableBalancesForPayables {
     fn skeleton_opt(&self) -> Option<ResponseSkeleton> {
         self.response_skeleton_opt
     }
 }
 
 #[derive(Clone, PartialEq, Eq, Debug, Message)]
-pub struct SetDbPasswordMsg {
-    pub client_id: u64,
-    pub password: String,
+pub struct ReportAccountsPayable {
+    pub accounts: Vec<PayableAccount>,
+    pub response_skeleton_opt: Option<ResponseSkeleton>,
 }
 
-#[derive(Clone, PartialEq, Eq, Debug, Message)]
-pub struct SetGasPriceMsg {
```

Comment:
Same with this, I don't understand why was it removed?

Reply:
Same as my comment above...